ui/inspectors/system-section.reel/mail.reel/mail.js

```diff
@@ -15,10 +17,19 @@ exports.Mail = Component.specialize(/** @lends Mail# */ {
         }
     },
 
+    encryptionOptions: {
+        value: null
+    },
+
+    templateDidLoad: {
+        value: function() {
+            this.encryptionOptions = MailEncryptionType.members;
```

Comment:
You should do the structure change ({lbal, value}) here instead of using a converter in the serialization.